PhysicsTools/Heppy/python/analyzers/objects/JetAnalyzer.py

```diff
@@ -126,6 +154,9 @@ def process(self, event):
         rho  = float(self.handles['rho'].product()[0])
         self.rho = rho
 
+        run=event.input.eventAuxiliary().id().run()
+        runBin=findBin(run, self.runsGT)
```

Comment:
replace with `runBin = bisect(self.runsGT, run) - 1` (and throw and exception if the runBin is -1 at the end)